src/store/mutations.js

```diff
@@ -274,5 +274,41 @@ export default {
   },
   SET_WATS_NEW_MODAL_VERSION (state, { version }) {
     state.watsNewModalVersion = version
+  },
+  TOOGLE_BLOCKCHAIN (state, { network, walletId, chainId, enable }) {
+    if (!state.enabledChains) {
+      Vue.set(state, 'enabledChains', {})
+    }
+    if (!state.enabledChains[walletId]) {
+      Vue.set(state.enabledChains, walletId, {})
+    }
+    if (!state.enabledChains[walletId]?.[network]) {
+      Vue.set(state.enabledChains[walletId], network, [])
+    }
```

Comment:
Can the 3 above be replaced with something like `ensureNetworkWalletTree` ?

Reply:
fixed